enterprise/config/config.go

```diff
@@ -23,6 +23,8 @@ type Config struct {
 	MinimumInstance int
 	// StorePath is the file path to store license.
 	StorePath string
+	// Mode is the application build mode, should be dev or release
+	Mode string
```

Comment:
Maybe use jwt aud? https://sourcegraph.com/github.com/bytebase/bytebase/-/blob/server/jwt.go?L82:9

Reply:
The mode here is used to make sure we only allow unlimited expires time under the dev environment. But put it into aud is a good point to do the validation check by different environment, I think I can use this in another PR (need to update hub code)

Reply:
I guess for dev mode, we can generate a license with distant future expires time (which is essentially no expire time). Thus we don't need additional code

Reply:
Now update with mock license with 999 years expires time 🤪